OMPython/__init__.py

```diff
@@ -447,3 +462,727 @@ def getClassNames(self, className=None, recursive=False, qualified=False, sort=F
+		self.changable = changable
+		self.description = description
+		self.variability = variability
+		self.causality = causality
+		
+#author = Sudeep Bajracharya
+#sudba156@student.liu.se
+#LIU(Department of Computer Science)
+class ModelicaSystem(object):
+
+	def __init__(self, fileName = None, modelName = None, lmodel = None):
+		if fileName is None and modelName is None and lmodel is None:
+			self.getconn = OMCSession()
+			return
+		
+		if fileName is None:
+			return "File does not exist"
+		
+		self.tree = None
+		self.quantitiesList = []
+		self.parList = []
+		self.parVarList = []
+		self.qNamesList = []
+		self.cNamesList = []
+		self.cValuesList = []
+		self.iNamesList = []
+		self.iValuesList = []
+		self.inputs = []
+		self.inputsVal = []
+		self.oNamesList = []
+		self.pNamesList = []
+		self.pValuesList = []
+		self.oValuesList = []
+		self.simNamesList = ['startTime', 'stopTime', 'stepSize', 'tolerance', 'solver']
+		self.simValuesList = []
+		self.optimizeOptionsNamesList = ['startTime', 'stopTime', 'numberOfIntervals', 'stepSize', 'tolerance', 'simflags']
+		self.optimizeOptionsValuesList = ['0.0', '1.0', '500', '0.002','1e-8',' ']
+		self.linearizeOptionsNamesList = ['startTime', 'stopTime', 'numberOfIntervals', 'stepSize', 'tolerance', 'simflags']
+		self.linearizeOptionsValuesList = ['0.0', '1.0', '500', '0.002','1e-8',' ']
+		self.getconn = OMCSession()
+		self.pwd = None
+		self.buildModelResult = None
+		self.xmlFile = None
+		self.lmodel = lmodel
+		self.modelName = modelName
+		self.fileName = fileName
+		self.simFlag = False
+		self.inputFlag = False
+		self.csvFile = ''
+
+		if os.path.exists(self.fileName) == False:
+			print "Error: File does not exist!!!"
+			return
+
+		(head, tail) = os.path.split(self.fileName)
+		self.currDir = os.getcwd()
+		self.modelDir = head
+		self.fileName_ = tail
+
+		if(self.modelDir == ""):
+			file_ = os.path.exists(self.fileName_)
+			if(self.fileName_):
+				self.loadingModel(self.fileName_, self.modelName, self.lmodel)
+
+				return
+			else:
+				print "Error: File does not exist!!!"
+				return
+
+		else:
+			os.chdir(self.modelDir)
+			file_ = os.path.exists(self.fileName_)
+			print file_
+			self.model = self.fileName_[:-3]
+			if(self.fileName_):
+				os.chdir(self.currDir)
+				self.loadingModel(self.fileName, self.modelName, self.lmodel)
+			else:
+				print "Error: File does not exist!!!"
+				return
+
+				self.create_scalarVariableList()
+				return
+
+	def loadingModel(self, fName, mName, lmodel):
+		loadfileError = ''
+		loadfileResult = self.requestApi("loadFile", fName)
+		loadfileError = self.requestApi("getErrorString")
+		if loadfileError:
+			specError = 'Parser error: Unexpected token near: optimization (IDENT)'
+			if specError in loadfileError:
+				self.requestApi("setCommandLineOptions", '"+g=Optimica"')
+				self.requestApi("loadFile", fName)
+			else:
+				print 'loadFile Error: ', loadfileError
+			return
+			
+		if lmodel is None:
+			pass
```

Comment:
is this `pass` some leftover section placeholder yet-to-be-implemented? if not, why not make this clause `if lmodel:` instead of the weird if None pass + else construct?